tendrl/commons/utils/log_utils.py

```diff
@@ -0,0 +1,31 @@
+""" Handles logging functionality. """
+import sys
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+
+
+def log(log_priority,publisher_id,**log_payload):
```

Comment:
Getting all the parameters one by one is quite naive although I have used that for now, but I am working on more generic way of solving the issue, will update soon